src/util/compilationDatabase.ml

```diff
@@ -68,17 +68,20 @@ let load_and_preprocess ~all_cppflags filename =
             preprocess_command
```

Comment:
The same missing `-o` logic also needs to be in the `command` case above.

Reply:
I now added the -o logic also for the command case and tested it with `compiledb --command-style make` for https://github.com/goblint/bench/issues/17.